packages/KTable/KTable.vue

```diff
@@ -43,153 +43,154 @@
-            @click="column.sortable && $emit('sort', column.key, sortOrder)"
-          >
-            <slot
-              :name="`column-${column.key}`"
-              :column="column">
-              <span
-                :class="{'sr-only': column.hideLabel}"
-              >
-                {{ column.label ? column.label : column.key }}
+    class="k-table-wrapper"
+    @scroll.passive="scrollHandler"
+  >
+    <table
+      :class="{'has-hover': hasHover, 'is-clickable': isClickable, 'side-border': hasSideBorder}"
+      class="k-table">
+      <thead :class="{ 'is-scrolled': isScrolled }">
+        <tr :class="{ 'is-scrolled': isScrolled }">
+          <template>
+            <th
+              v-for="(column, index) in tableHeaders"
+              :key="index"
+              :class="{
+                'sortable': !column.hideLabel && column.sortable,
+                'active-sort': !column.hideLabel && column.sortable && column.key === sortColumnKey,
+                [sortColumnOrder]: column.key === sortColumnKey && !column.hideLabel,
+                'is-scrolled': isScrolled
+              }"
+              @click="column.sortable && sortClickHandler(column.key)"
+            >
+              <span class="d-flex align-items-center">
+                <slot
+                  :name="`column-${column.key}`"
+                  :column="column">
+                  <span
+                    :class="{'sr-only': column.hideLabel}"
+                  >
+                    {{ column.label ? column.label : column.key }}
+                  </span>
+                </slot>
+                <KIcon
+                  v-if="!column.hideLabel && column.sortable"
+                  class="caret ml-2"
+                  color="var(--KTableColor, var(--black-70, color(black-70)))"
+                  width="12"
+                  height="12"
+                  icon="chevronDown"
+                />
               </span>
-            </slot>
-          </th>
-        </template>
-      </tr>
-    </thead>
-    <tbody>
-      <tr
-        v-for="(row, rowIndex) in options.data"
-        :key="rowIndex"
-        v-bind="rowAttrs(row)"
-        v-on="trlisteners(row, 'row')">
-        <template>
-          <td
-            v-for="(value, index) in options.headers"
-            :key="index"
-            v-bind="cellAttrs({ headerKey: value.key, row, rowIndex, colIndex: index })"
-            v-on="tdlisteners(row[value.key], 'cell')">
-            <slot
-              :name="value.key"
-              :row="row"
-              :rowKey="rowIndex"
-              :rowValue="row[value.key]">{{ row[value.key] }}</slot>
-          </td>
-        </template>
-      </tr>
-    </tbody>
-  </table>
+            </th>
+          </template>
+        </tr>
+      </thead>
+      <tbody>
+        <tr
+          v-for="(row, rowIndex) in data"
+          :key="rowIndex"
+          v-bind="rowAttrs(row)"
+          v-on="trlisteners(row, 'row')"
+        >
+          <a
```

Comment:
I understand the intent here is to make it more accessible, but this is illegal html

Reply:
Agree -- good catch

Reply:
Fixed!

Reply:
We now are setting `tabindex=0`, `cursor: pointer`, and `role: link` if any listeners are defined on `<tr>` elems.